mlflow/metrics/utils/make_genai_metric.py

```diff
@@ -210,7 +226,7 @@ def score_model_on_one_payload(
                     )
                 )
 
-            for future in as_completed(futures):
+            for future in as_completed(futures, timeout=15):
```

Comment:
Can users change the `max_workers` via params or something? Since we don't know the rate limit for the models that users pass right? 🤔

Reply:
Yes good idea.